### PR TITLE
DOC: add doctr

### PR DESCRIPTION
**Pull request by CJ-Wright**
May need to activate Travis?

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Closes #37

**Comment by CJ-Wright** [comment]
Cycle for travis.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - python setup.py install
+
+script:
+  - set -e
+  - # Command to build your docs
+  - pip install doctr
+  - cd docs
+  - make html
+  - cd ..
+  - doctr deploy --deploy-repo ergs/rever-docs docs/_build/html
```
Comment (scopatz) [inline .travis.yml]: I think there is a bug in doctr with respect to explicitly putting in the sphinx dir.  I think this might need to be a `.`

**Comment by asmeurer** [comment]
Now codecov is failing.

**Comment by CJ-Wright** [comment]
Sorry, codecov shouldn't have been there in the first place :disappointed:

**Comment by asmeurer** [comment]
It worked https://github.com/ergs/rever-docs/commit/e5412e827480da1ae783dc0d2fb848ac25f9bc19 http://www.ergs.sc.edu/rever-docs/

**Comment by CJ-Wright** [comment]
yey! thank you!

**Comment by scopatz** [comment]
Awesome!

**Pull request merged**